ansible/roles/nginx/tasks/main.yml

```diff
@@ -7,7 +7,10 @@
 #
 
 - name: install
```

Comment:
nitpick: seeing how we're going from only installing nginx to installing multiple packages perhaps change to "install required packages"